core/chain_makers.go

```diff
@@ -421,14 +421,6 @@ func GenerateChain(config *params.ChainConfig, parent *types.Block, engine conse
 			b.header.RequestsHash = &reqHash
 		}
 
-		if config.IsDAFootprintBlockLimit(b.header.Time) {
-			gasUsed, err := types.CalcGasUsedJovian(b.txs, b.header.GasUsed)
-			if err != nil {
-				panic(err)
-			}
-			b.header.GasUsed = gasUsed
-		}
-
```

Comment:
Why don't we need to replace this code with something setting the header `blobGasUsed` field instead?

Reply:
Got it, thanks. I think I was more focussed on the other comment which seems a little less obvious.